daemon/daemon/k8s_watcher.go

```diff
@@ -279,36 +309,62 @@ func (d *Daemon) syncLB() {
 				}
 				svcPort.ServiceID = svcl4ID.ServiceID
 			}
+			nSvcs := uniqSvcs[svcPort.Port]
+			uniqSvcs[svcPort.Port] = nSvcs - 1
+
 			isServerPresent := false
 
-			serverIndex := 0
+			serverIndex := uint16(0)
 
 			if len(se.IPs) != 0 {
-				log.Debugf("# cilium lb update-service %s %d %d %d %d %s %d",
-					svcInfo.IP, svcPort.Port, serverIndex, len(svcInfo.Ports), svcPort.ServiceID,
-					"::", 0)
+				svc6k := lbmap.NewService6Key(svcInfo.IP, svcPort.Port, serverIndex)
```

Comment:
Yes, I will change this a bit, probably on a new commit. I detect if the IP is IPv4 or IPv6 and add it on the respective service map